### ISSUE TITLE
Support different namespaces on different extern blocks

### ISSUE THREAD

**Issue by dtolnay** [dtolnay/cxx#166]
Currently the namespace is determined globally for an entire #\[cxx::bridge\] invocation.

```rust
#[cxx::bridge(namespace = path::to)]
mod ffi {...}
```

It would be good to be able to pull in things from multiple different namespaces at once. Something like:

```rust
#[cxx::bridge]
mod ffi {
    #[namespace = path::to::first]
    extern "C" {
        type First;
    }

    #[namespace = path::to::second]
    extern "C" {
        type Second;
    }

    #[namespace = path::to::third]
    extern "Rust" {
        fn wow(first: &First, second: &Second);
    }
}
```

**Comment by dtolnay** [comment]
Implemented as part of #370.

**Comment by dzhang-b** [comment]
if i have symbols from C header, how should I declare them in cxx? I assume they are in global namespace, but is not in `std` namespace.  Currently it reports `my_target_t’ in namespace ‘std’ does not name a type`
